tests/code/test_emailHandler.py

```diff
@@ -50,65 +47,65 @@ def test_send_email_using_modal():
 
                 transaction.rollback()
 
-@pytest.mark.integration
-@pytest.mark.skip(reason="Authentication issues")
-def test_sending_automated_email():
-    with app.test_request_context():
-        with mainDB.atomic() as transaction:
-            # Case 2: Send email without subject and body -- as if email is sent automatically
-            url_domain = urlparse(request.base_url).netloc
-            raw_form_data = {"templateIdentifier": "Test",
-                "programID":"1",
-                "eventID":"1",
-                "recipientsCategory": "Interested"}
+# @pytest.mark.integration
+# def test_sending_automated_email():
+#     with app.test_request_context():
+#         with mainDB.atomic() as transaction:
+#             # Case 2: Send email without subject and body -- as if email is sent automatically
+#             url_domain = urlparse(request.base_url).netloc
+#             raw_form_data = {"templateIdentifier": "Test",
+#                 "emailSender": "neillz"
+#                 "eventID":"1",
+#                 "recipientsCategory": "Interested"}
 
-            email = EmailHandler(raw_form_data, url_domain, User.get_by_id("neillz"))
+#             email = EmailHandler(raw_form_data, url_domain)
 
-            with email.mail.record_messages() as outbox:
-                email_sent = email.send_email()
-                assert email_sent == True
+#             with email.mail.record_messages() as outbox:
+#                 email_sent = email.send_email()
+#                 assert email_sent == True
 
-                assert len(outbox) == 2
-                assert outbox[0].subject == "Test Email"
-                assert outbox[0].body == "Hello Sreynit Khatt, This is a test event named Empty Bowls Spring Event 1 located in Seabury Center. Other info: 10/12/2021-06/12/2022 and this 06:00-09:00."
+#                 assert len(outbox) == 2
+#                 assert outbox[0].subject == "Test Email"
+#                 assert outbox[0].body == "Hello Sreynit Khatt, This is a test event named Empty Bowls Spring Event 1 located in Seabury Center. Other info: 10/12/2021-06/12/2022 and this 06:00-09:00."
 
-                transaction.rollback()
+#                 transaction.rollback()
```

Comment:
Why have you commented out a whole test? If it is no longer needed it should be deleted.

Reply:
Okay, sniped it